modules/lm-coursier/src/main/scala/lmcoursier/definitions/ToCoursier.scala

```diff
@@ -36,6 +35,27 @@ object ToCoursier {
       module.attributes
     )
 
+  def moduleMatcher(matcher: ModuleMatcher): coursier.util.ModuleMatcher =
+    if (matcher == ModuleMatcher.all) coursier.util.ModuleMatcher.all
+    else coursier.util.ModuleMatcher(module(matcher.matcher))
+
+  def moduleMatchers(matcher: ModuleMatchers): coursier.util.ModuleMatchers =
+    if (matcher == ModuleMatchers.all) coursier.util.ModuleMatchers.all
```

Comment:
Same as above, that `if` should be needed I think.

Reply:
6355338735046930e5af448ef3f3a976e98c548d